### DISCUSSION TITLE
Do not collapse sample point indicators into dots if there is enough room to display them

### DISCUSSION THREAD

**Discussion by rust2** [ppy/osu#31140] [Ideas]
### Type

Game behaviour

### Bug description

See https://github.com/ppy/osu/pull/29896.
Collapse should happen for only indicators which actually need it and not for all visible in case if there is not enough space for some of them and they become collapsed.
Sorry for bad english but as you can see on image, not all indicators need to collapse

### Screenshots or videos

![osu_2024-12-09_22-02-32](https://github.com/user-attachments/assets/e41e95a0-a20e-405c-a6e7-574b5e1f214e)
![osu_2024-12-09_22-17-49](https://github.com/user-attachments/assets/3bf99a4a-aa1a-4ac8-836f-ff08fe517668)

### Version

2024.1208.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18065262/compressed-logs.zip)

**Comment by bdach** [comment]
Not sure I agree with this one.

**Reply by peppy** [reply]
Also not sure. Needs some further thought / discussion.
